onnx/defs/rnn/defs.cc

```diff
@@ -54,7 +55,6 @@ std::function<void(OpSchema&)> RNNDocGenerator(const char* name) {
     };
```

Comment:
I wonder whether we should make output Y_h optional too for symmetry purposes. You can probably sneak it in without bumping the version as it's just an optional output and optionals are not yet properly handled everywhere anyway.

Reply:
I am fine with this. Should we go even further to make the outputs of every op that has >1 outputs optional?